source/custom/niscope_service.custom.cpp

```diff
@@ -564,4 +564,18 @@ ::grpc::Status NiScopeService::ReadMeasurement(::grpc::ServerContext* context, c
   }
 }
 
+::grpc::Status NiScopeService::ConvertApiErrorStatusForViSession(google::protobuf::int32 status, ViSession vi)
```

Comment:
I'm marking it here because I know scope has a lot of custom service handlers. But we'll need to go through and update all of them across the _service.custom.cpp files so their status checks call the new ConvertApiErrorStatusForViSession.

This can be done in this change or in a separate one.

Reply:
Done in this PR. @reckenro if you don't mind, please take another look at those.